templates/engage/mir-for-iot.html

```diff
@@ -0,0 +1,49 @@
+{% extends_with_args "engage/base_engage.html" with title="Build smart display devices with Mir Whitepaper" meta_image="https://assets.ubuntu.com/v1/8b2a5baf-61620202-4f636100-ac68-11e9-9c88-ffdccb58ecdf.jpg" meta_description="Mir is an open-source library that simplifies the creation of graphical interfaces for secure, smart display devices." %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/11E9O_mVN_U63ZLsZOvwsdXAyNtpRv0aF8_VWBDfrnRc{% endblock meta_copydoc %}
+
+{% block content %}
+
+{% include "engage/shared/_header.html" with title="Build smart display devices with Mir: fast to production, secure, open-source" url="#the-form" cta="Download the whitepaper" class="p-strip--dark p-takeover--mir"%}
```

Comment:
```suggestion
{% include "engage/shared/_header.html" with title="Build smart display devices with Mir: fast to production, secure, open-source" url="#the-form" cta="Download the whitepaper" class="p-strip--dark p-takeover--mir" %}
```
Just adding a space for tidier code